docs/intro.mdx

```diff
@@ -59,7 +60,9 @@ df["input length"] = df["text"].str.len()
 Let's create a project for this task. **Projects** in Zeno are a base dataset and any number of AI system outputs, and are used to evaluate and compare model performance. Here we create a project and upload our base dataset.
 
 :::info
-If you are working with tabular data you do not have to pass in a `view` option to `create_project` or a `data_column` option to `upload_dataset`.
+The `view` option can take a [`string`](/docs/views/existing) for one of the standard views or a [`dict`](http://localhost:3000/docs/views/spec/) with a custom view specification.
```

Comment:
Does this work? Shouldn't the localhost part be removed here?

Reply:
oop good catch